GoogleSignIn/Sources/GIDAppAuthFetcherAuthorizationWithEMMSupport.m

```diff
@@ -0,0 +1,127 @@
+// Copyright 2022 Google LLC
```

Comment:
Use a block comment for the copyright notice, consistent with the other files you're adding here.